content/shared/influxdb3-cli/delete/database.md

```diff
@@ -11,7 +11,7 @@ influxdb3 delete database [OPTIONS] <DATABASE_NAME>
 
 ## Arguments
 
-- **DATABASE_NAME**: The name of the database to delete.
+- **DATABASE_NAME**: The name of the database to delete. Valid database names are alphanumeric and start with a letter or number. Dashes (`-`) and underscores (`_`) are allowed.
```

Comment:
Name requirements are really only necessary for `create`, b/c you need to know what chars you can use. For `delete`, you just need to know the existing database name.  However, if we don't already mention the (token) permissions required, we should add that.